livekit-agents/livekit/agents/job.py

```diff
@@ -303,6 +307,11 @@ def add_sip_participant(
             trunk_id: The ID of the SIP trunk to use
             participant_identity: The identity of the participant to add
             participant_name: The name of the participant to add
+            wait_until_answered: Blocks until user answers the call, or if the call fails
```

Comment:
maybe use the same language as https://docs.livekit.io/sip/api/#createsipparticipant - this blocks until answered by a user _or_ an automated system